### DISCUSSION TITLE
Dashboard

### DISCUSSION THREAD

**Discussion by sschneider-ihre-pvs** [lost-pixel/lost-pixel#369] [Q&A]
is there a dashboard for the on-premise usage of any kind to have some slider to partially watch differences or something?

**Comment by d-ivashchuk** [comment]
I was working on Lost Pixel Studio, some time ago but then priorities shifted. Studio would allow you almost our platform experience when it comes to viewing the difference without most of the collaboration features. If you would be interested, I might as well invite you to help me finish it up.

it was already in pretty good shape, but still not ready for public release & usgae

**Reply by sschneider-ihre-pvs** [reply]
sure, after the histoire PR :D
